app/controllers/course/discussion/posts_controller.rb

```diff
@@ -0,0 +1,106 @@
+class Course::Discussion::PostsController < Course::ComponentController
+  before_action :set_topic
```

Comment:
Oh, I'm not sure what is the best way to do it in Rails but I want to set `@conditional` variable before each action. I use `set_conditional` method to do it. Not sure how to do it using a getter.